tools/c7n_tencentcloud/c7n_tencentcloud/provider.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
we need both spdx license headers on all python files.

Reply:
Do you mean both line of the below? 
"# Copyright The Cloud Custodian Authors.
# SPDX-License-Identifier: Apache-2.0"

Reply:
ok

Reply:
done
commit: [e7ff155](https://github.com/cloud-custodian/cloud-custodian/pull/7688/commits/e7ff1551bc93a998fb3b23d3808042eaac9d976d)